components/package.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
We also need to update the dependencies and peerDependencies in this file :)

Reply:
you are right 👍